### ISSUE TITLE
Create spreadsheet and upload to cloud

### ISSUE THREAD

**Issue by lasmacdonald** [lasmacdonald/samples-lasmacdonald#1]
@larnsce @mianzg @sskorik01

**Comment by lasmacdonald** [comment]
I completed homework assignment 04. I committed and pushed, although I'm not sure I was supposed to.

To the detriment of my application of what we learned in class, I had already done quite a few of the recommendations for spreadsheets, so I did not have much modification to do. But, a few questions:

- I tried to use gsub to remove the 00:00:00 from my sample dates but wasn't able to do so successfully. any tips? this is what I followed: https://www.perplexity.ai/search/how-do-you-UVxLFGf1T8ujvvzg5vANMA?s=c
- From the example in the HW assignment, I think I may have chosen values far too low for the septic tanks (https://www.perplexity.ai/search/what-is-the-T43JGMa0RwukcW37AKBv5Q?s=c told me they're typically in mg/L as compared to g/L for pit latrines). This made me wonder, if I wanted to only update the values for septic tanks in my spreadsheet (say, by multiplying by 1000), how would I do so?

**Comment by larnsce** [comment]
Yes, thank you for doing the commit & push. I had missed that in the task description.

You had two really good questions which came in time while I was developing the lecture for the module. I hope it's okay that I have your examples in the live coding session.

If you get a chance, let me know if you could try it out in your code and if it worked.

**Comment by larnsce** [comment]
When you create the next repo, please do that on the GitHub organisation for the course. https://github.com/ds4owd

You opened this one on your personal profile, which is also a great place, especially after the course, but keeping it all at the ds4owd GitHub organisation helps us manage everything in one place.

**Comment by lasmacdonald** [comment]
Noted! Will be sure to do so.

And no problem. Thanks for answering my questions. Seeing them in class only made me wish I hadn't given my file such a silly name! I will try the code out before next Tuesday's class and let you know here, then close out this issue.
